Server/Application/CashByCodeMethods/Command/GenerateCashByCodeCommand.cs

```diff
@@ -65,12 +68,21 @@ public async Task<CommandResult> Handle(GenerateCashByCodeCommand command , Canc
 
         private async Task LifeSpan(Guid Id,IServiceScopeFactory serviceScopeFactory)
         {
-            await Task.Delay(3600000);
+            await Task.Delay(100000);
             using var scope = serviceScopeFactory.CreateScope();
             var dbContext = scope.ServiceProvider.GetService<ApplicationDbContext>();
-            var itemToRemove = dbContext.CashByCodeRepo.FirstOrDefault(e => e.Id == Id);
-            dbContext.CashByCodeRepo.Remove(itemToRemove);
-            await dbContext.SaveChangesAsync();
+            if (dbContext.CashByCodeRepo.Any(e => e.Id == Id))
+            {
+                var itemToRemove = dbContext.CashByCodeRepo.FirstOrDefault(e => e.Id == Id);
```

Comment:
Instead of doing `CashByCodeRepo.Any()` then `CashByCodeRepo.FirstOrDefault()` you could simply call `CashByCodeRepo.FirstOrDefault()` and see if the returned item is null. So that you're not making 2 calls to the database